ObjectiveSee/OverSight.pkg.recipe

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Description</key>
+    <string>Downloads, unzips, verifies the signature and packages the installer of the latest version of OverSight by Objective-See.</string>
+    <key>Identifier</key>
+    <string>com.github.its-unibas.pkg.OverSight</string>
+    <key>MinimumVersion</key>
+    <string>1.0.0</string>
+    <key>Input</key>
+    <dict>
+        <key>NAME</key>
+        <string>OverSight</string>
+    </dict>
+    <key>ParentRecipe</key>
+    <string>com.github.its-unibas.download.OverSight</string>
+    <key>Process</key>
+    <array>
+        <dict>
+            <key>Arguments</key>
+            <dict>
+                <key>pkgdirs</key>
+                <dict>
+                    <key>usr/local/var/oversight</key>
+                    <string>0775</string>
+                </dict>
+                <key>pkgroot</key>
+                <string>%RECIPE_CACHE_DIR%/%NAME%/pkg</string>
+            </dict>
+            <key>Processor</key>
+            <string>PkgRootCreator</string>
+        </dict>
+        <dict>
+            <key>Processor</key>
+            <string>FileMover</string>
+            <key>Arguments</key>
+            <dict>
+                <key>source</key>
+                <string>%RECIPE_CACHE_DIR%/%NAME%/%NAME% Installer.app</string>
```

Comment:
Same as obove. Don't bother with the `.pkg` recipe if only the `installer.app` is installed.